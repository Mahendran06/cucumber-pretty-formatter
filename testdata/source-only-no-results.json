{"event": "SourceRead", "feature_source_id": "features/my.feature:1", "source": "@hello\nFeature: My feature\n  Scenario: My scenario\n    Given not run"}
