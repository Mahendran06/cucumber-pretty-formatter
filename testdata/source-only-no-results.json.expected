@hello
Feature: My feature
  Scenario: My scenario
    Given not run
